packages/core/prompt/BasePromptSession.ts

```diff
@@ -408,6 +408,10 @@ export abstract class BasePromptSession {
 				}
 			}
 
+			if (config.project.framework === "react") {
```

Comment:
This is how we used to do it, yes. However, now that we are using `eslint` having an `extend` option in the `package.json` causes it to attempt to extend the provided modules. These modules are part of the templated projects and even though I've specified that we strictly want to ignore directories under the templates, `eslint` still goes over them and looks at their configs. And since the package is nowhere to be found (as you'd expected from the templates' dependencies) `eslint` throws. So, I managed to work around this by adding this option dynamically when the project structure is present. I've described this briefly in the PR's description but maybe I should've been more thorough. :smile: